Parse/PFFile.h

```diff
@@ -152,6 +154,11 @@ NS_ASSUME_NONNULL_BEGIN
 ///--------------------------------------
 
 /**
+ Sets a custom upload controller that synchronously uploads the file using its own policy.
+ */
+@property (nonatomic, strong, readwrite, nullable) id<PFFileUploadController> uploadController;
```

Comment:
I'd put the controller part of the configuration to avoid mismatches, keep it there as well to use the default uploader for a particular file no?